docs/testing.md

```diff
@@ -311,9 +311,9 @@ This will run the linter, Shellcheck, audit, build, test, Cypress, Cerebral test
 
 ## PDF Testing
 
-Since our system generates a lot of PDFs, we have a set of tests that verify the pdfs didn't change using a checksum of the first exported image of the pdfs.  Since all of these PDFs share a single .scss file, there is risk involved when trying to update a single PDF to accidentally change the styles of other PDFs.  Therefore, we have a set of tests that verify that the PDFs are not changing.
+Since our system generates a lot of PDFs, we have a set of tests that verify the PDFs didn't change using a checksum of the first exported image of the PDFs.  Since all of these PDFs share a single .scss file, there is risk involved when trying to update a single PDF to accidentally change the styles of other PDFs.  Therefore, we have a set of tests that verify that the PDFs are not changing.
 
-If you are adding/creating a new PDF you must add the PDF name to the pdfs array in image-compare-pdfs.js.
+If you are adding/creating a new PDF you must add the PDF name to the PDF array in image-compare-pdfs.js.
```

Comment:
This one is referring to  an array called `pdfs`. Maybe just throw it in some backticks.
```suggestion
If you are adding/creating a new PDF you must add the PDF name to the `pdfs` array in image-compare-pdfs.js.
```

Reply:
Thank you!